rp/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "rp"
+version = "0.2.1"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+description = "Build post-quantum-secure VPNs with WireGuard!"
+homepage = "https://rosenpass.eu/"
+repository = "https://github.com/rosenpass/rosenpass"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
+rosenpass = { workspace = true }
+rosenpass-ciphers = { workspace = true }
+rosenpass-cipher-traits = { workspace = true }
+rosenpass-secret-memory = { workspace = true }
+rosenpass-util = { workspace = true }
+
+tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
+wireguard-keys = "0.1"
```

Comment:
I am skeptical about the `wireguard-keys` crate. It has not been updated in two years (which is not a bad thing per se, I believe that software can be finished). However, their GitLab repository is private, or at least I am unable to access is, which is not unacceptable for a cryptographic crate IMO.

Reply:
Resolved in ca8a587b6fd07f49ebb4ba6dce831d72e61d415a.